core/Cargo.toml

```diff
@@ -27,8 +27,8 @@ smallvec = "1.6.1"
 url = { version = "2.2.0", features = ["serde"] }
 
 [[example]]
-name = "http_bench_bin_ops"
-path = "examples/http_bench_bin_ops.rs"
+name = "http_bench_buffer_ops"
+path = "examples/http_bench_buffer_ops.rs"
```

Comment:
Can you please keep previous names? Otherwise benchmark page might break or lose results for previous examples.

Reply:
Maybe in general, instead of "buffer ops", "bin ops" should be used as name? What do you think?

Reply:
@inteon I think that's a good idea, after all we're exchanging binary encoded data :) once you're done with renaming I think it's good to land